community templates/README.md

```diff
@@ -0,0 +1,84 @@
+# Community pipeline templates #
+
+Pipeline templates should follow ARM templates syntax. For more information about ARM templates, see [https://go.microsoft.com/fwlink/?linkid=2181481](https://go.microsoft.com/fwlink/?linkid=2181481)
+
+Notes for "manifest.json":
+1. It must contain a property named "contributorType" set to "Community".
+2. "author" property can be configured as the contributor of the template.
+
+# Create templates from pipelines #
+
+In addition to writing a template file directly, you can also get a template from a pipeline in [Azure Data Factory](https://adf.azure.com). 
+For details about pipelines please refer [Data Integration Pipelines](https://go.microsoft.com/fwlink/?linkid=2181927).  
+
+To create a template from a pipeline, please go to [Azure Data Factory](https://adf.azure.com) and excute the following steps:
+
+1. Navigate to your data factroy, in the left side menu choose "Author" to open the authoring blade.
+
+   ![Open authoring pane](images/open-authoring-pane01.png?raw=true)
+
+2. Set up your [code repository](https://go.microsoft.com/fwlink/?linkid=2181834) if a repository is not configured in your data factory or switch to your existing configured git repository beacuse creating a template from a pipeline is only enabled in git mode.
```

Comment:
Seems 7 has nothing to do with other steps. Can't we export it directly? 
For step 7, maybe we should ask them to go to their git repo, add the properties we need

Reply:
Updated